integration-tests/tests/aws/get-ivq-questions/get-ivq-questions-unhappy.test.ts

```diff
@@ -0,0 +1,54 @@
+import { stackOutputs } from "../resources/cloudformation-helper";
+import { clearItems } from "../resources/dynamodb-helper";
+import { executeStepFunction } from "../resources/stepfunction-helper";
+import {
+  getSSMParamter,
+  ssmParamterUpdate,
+} from "../resources/ssm-param-helper";
+
+describe("get-ivq-questions-unhappy", () => {
+  const stateMachineInput = {
+    sessionId: "12346",
+    nino: "AA000003D",
+  };
+
+  let output: Partial<{
+    QuestionsTable: string;
```

Comment:
Is `QuestionsTable` needed? If you removed this, will it break? If not, let's remove this 1 line.